_search-plugins/cross-cluster-search.md

```diff
@@ -243,3 +243,56 @@ curl -XGET -k -u booksuser:password 'https://localhost:9250/opensearch-ccs-clust
   }]
 }
 ```
+
+## Sample bare metal/virtual machines setup
+
+If you are running OpenSearch on a bare metal server or using a virtual machine, you can run the same commands, specifying the IP (or domain) of the OpenSearch cluster.
+For example, in order to configure remote cluster for Cross-cluster search, find the IP of the remote node or domain of the remote cluster and run command:
```

Comment:
Either combine with the previous sentence or insert a line break. Should we tell/link the reader where they can find the IP or the domain?